<?xml version="1.0" encoding="UTF-8"?><testrun name="TestRunner" project="E2EProjct" tests="5" started="5" failures="0" errors="2" ignored="4">
  <testsuite name="cucmberOptions.TestRunner" time="0.039">
    <testsuite name="Feature: Login into Application" time="0.039">
      <testsuite name="Scenario: Positive test validation login" time="0.032" incomplete="true">
        <error>cucumber.runtime.CucumberException: Failed to instantiate class stepDefinations.stepDefination&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:47)&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.getInstance(DefaultJavaObjectFactory.java:33)&#13;
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:38)&#13;
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)&#13;
	at cucumber.runtime.Runtime.runStep(Runtime.java:300)&#13;
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)&#13;
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)&#13;
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)&#13;
	at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:102)&#13;
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)&#13;
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)&#13;
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:95)&#13;
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:38)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at cucumber.api.junit.Cucumber.run(Cucumber.java:100)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
Caused by: java.lang.reflect.InvocationTargetException&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#13;
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:41)&#13;
	... 32 more&#13;
Caused by: java.lang.NullPointerException&#13;
	at resources.base.&lt;init&gt;(base.java:23)&#13;
	at stepDefinations.stepDefination.&lt;init&gt;(stepDefination.java:18)&#13;
	... 38 more&#13;
&#13;
</error>
        <testcase name="Given Intialize the browser with &quot;chrome&quot;" classname="Scenario: Positive test validation login" time="0.032">
          <error>cucumber.runtime.CucumberException: Failed to instantiate class stepDefinations.stepDefination&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:47)&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.getInstance(DefaultJavaObjectFactory.java:33)&#13;
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:38)&#13;
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)&#13;
	at cucumber.runtime.Runtime.runStep(Runtime.java:300)&#13;
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)&#13;
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)&#13;
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)&#13;
	at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:102)&#13;
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)&#13;
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)&#13;
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:95)&#13;
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:38)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at cucumber.api.junit.Cucumber.run(Cucumber.java:100)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
Caused by: java.lang.reflect.InvocationTargetException&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#13;
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)&#13;
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:41)&#13;
	... 32 more&#13;
Caused by: java.lang.NullPointerException&#13;
	at resources.base.&lt;init&gt;(base.java:23)&#13;
	at stepDefinations.stepDefination.&lt;init&gt;(stepDefination.java:18)&#13;
	... 38 more&#13;
&#13;
</error>
        </testcase>
        <testcase name="And Navigate to &quot;https://rahulshettyacademy.com&quot; site" classname="Scenario: Positive test validation login" time="0.001" ignored="true"/>
        <testcase name="And Click on login link in home page to land on secure sign in page" classname="Scenario: Positive test validation login" time="0.0" ignored="true"/>
        <testcase name="When User enters &quot;test99@gmail.com&quot; and &quot;123456&quot; and logs in" classname="Scenario: Positive test validation login" time="0.0" ignored="true"/>
        <testcase name="Then Verify that the user is successfully logged in" classname="Scenario: Positive test validation login" time="0.004" ignored="true"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testrun>
